index.html

```diff
@@ -0,0 +1,270 @@
+        </div>
+        <div class="video-info">
+          <div id="info">
+            <h1>Is The New 2020 Ducati Superleggera V4 Worth €100,000?</h1>
+          </div>
+          <div id="info-1">
+            <div id="info-text">
+              <p class="info-p">480,905 views . 5 March 2020 </p>
+            </div>
+            <div id="top-buttons">
+              <a href="">
+                <i class="fas fa-thumbs-up"></i>
+              </a>
+              <p class="info-p">5560</p>
+              <a href="">
+                <i class="fas fa-thumbs-down"></i>
+              </a>
+              <p class="info-p">197</p>
+              <a href="">
+                <i class="fas fa-share"></i>
+              </a>
+              <p class="info-p">SHARE</p>
+              <a href="">
+                <i class="fas fa-square"></i>
+              </a>
+              <p class="info-p">SAVE</p>
+              <a href="">
+                <i class="fas fa-ellipsis-h"></i>
+              </a>
+            </div>
+          </div>
+        </div>
+        <hr>
+        <div id="chanel-info">
+          <div id="top-container-chanel-info">
+            <div id="video-owner">
+              <div>
+                <img id="img-owner" src="img/owner-img.jpg" alt="MOTOBOB Chanel">
+              </div>
+              <div id="inf-owner">
+                <h2>MOTOBOB</h2>
+                <p class="info-p">84.6K subscribers</p>
+              </div>
+            </div>
+            <div id="subscribe-button">
+              <div class="button">SUBSCRIBE</div>
+            </div>
+          </div>
+          <div id="chanel-description">
+            <h4>Sign up for my email newsletter of the best motorcycling stories here:</h4>
+            <a href="https://mailchi.mp/306039bd7519/motobob">
+              <span>https://mailchi.mp/306039bd7519/motobob</span>
+            </a>
+            <p class="info-p"><b><br><b><br>SHOW MORE</p>
+          </div>
+        </div>
+        <hr>
+        <b><br><b><br>
+            <div id="commments-seccion-principal">
+              <div id="comments-header">
+                <div id="title-comments">
+                  <p class="info-p1">800 Commmnets</p>
+                  <a href=""><i class="fas fa-align-left"></i></a>
+                  <p class="info-p">SORT BY</p>
+                </div>
+                <div id="write-comments">
+                  <img id="user-img" src="img/user.jpg" alt="User">
+                  &nbsp&nbsp&nbsp&nbsp&nbsp
+                  <div id="box-comment">
+                    <input id="commets-bar" type="text" name="comment" placeholder="Add a public comment...">
+                  </div>
+                  <div id="cc-button">
+                    <div id="button-cancel">CANCEL</div>
+                    <div id="button-comment">COMMENT</div>
+                  </div>
+                </div>
+              </div>
+              <b><br><b><br><b><br><b><br>
```

Comment:
- [Optional] Instead of repeating `<br>` repeatedly, use margin and padding in css files.